README.md

```diff
@@ -114,14 +113,14 @@ Using compiled files:
 ### Cleaning the build ###
     ./gradlew clean
 
-### Running a task with one of the Scala versions available (2.12.x or 2.13.x) ###
+### Running a task with Scala 2.13.x ###
```

Comment:
I think we should just delete this section and all the other sections related to multiple Scala versions, etc.

Reply:
I've kept the section related to ./gradlewAll since we haven't removed it yet.

Reply:
I’ve kept the section related to ./gradlewAll since we haven’t removed it yet.

Reply:
I would remove that too. We can simply add a comment to `gradlewAll` explaining that it has been kept for compatibility reasons, but there is no reason to use it anymore.